python/nav/web/report/views.py

```diff
@@ -292,18 +293,19 @@ def make_report(request, report_name, export_delimiter, query_dict, paginate=Tru
         ["%s=%s" % (x, y) for x, y in iteritems(query_dict) if x != 'page_number']
     )
 
+    config_files = list_config_files_from_dir(CONFIG_DIR)
+
     @report_cache(
         (
             request.account.login,
             report_name,
-            os.stat(CONFIG_FILE_PACKAGE).st_mtime,
-            os.stat(CONFIG_FILE_LOCAL).st_mtime,
+            [stat(path).st_mtime for path in config_files],
```

Comment:
Is there a way to do this in a for-loop or should it stay this way?

Reply:
I'd say this list comprehension is perfectly readable. Short and sweet, no conditionals, can't easily be reduced to `map()`.

And no, if you wanted to do it in a loop you'd have to run the loop elsewhere and stash the list in some variable or other. Clunky, in comparison.

Reply:
I think this is pretty succinct as well 👍